cfgov/jinja2/v1/_includes/macros/video-player.html

```diff
@@ -55,7 +55,8 @@
     {% set button_pos   = options.button_pos or 'center' %}
     {% set is_flexible  = ( options.video.width or options.video.height ) is not defined %}
     <div class="video-player
-                video-player__{{ video_player }}"
+                video-player__{{ video_player }}
+                {{ 'featured-content-module_visual' if is_flexible == false }}"
```

Comment:
My understanding would be if video-player is an organism and it has margin/float adjustments, those would be set in the template. If it's more complex changes, it would be a modifier, or an entirely different organism. 

The issue is that if the video-player organism can take arbitrary classes, it can morph into a unique snowflake that isn't a video-player organism anymore (as originally authored). That means it can't be guaranteed to be covered in documentation from design or be expected to behave the same way in tests as other video-player instances.